pkgs/by-name/im/immersed/linux.nix

```diff
@@ -2,8 +2,63 @@
 , version
 , src
 , meta
+, pkgs
+, lib
 , appimageTools
+, makeWrapper
+, libgpg-error
+, fontconfig
+, libSM
+, libGL
+, libz
+, mesa
+, wayland
+, pipewire
+, fribidi
+, harfbuzz
+, freetype
+, libthai
+, e2fsprogs
+, zlib
+, libp11
+, xorg
```

Comment:
```suggestion
, appimageTools
```

Reply:
resolved by: 172d66c38e22c0e56a2a9362963bdc692601eb1d

Reply:
Uh... This suggestion is missing including `libgpg-error` package...
```diff
--- a/pkgs/by-name/im/immersed/linux.nix
+++ b/pkgs/by-name/im/immersed/linux.nix
@@ -3,6 +3,7 @@
 , src
 , meta
 , appimageTools
+, libgpg-error
 }:

 let
```

Reply:
This problem is resolved by: cfac9c18427ca5ea0cb1d5fcfb8335f361bf90c1